swo_crm_service_client/client.py

```diff
@@ -0,0 +1,106 @@
+from dataclasses import dataclass
+from urllib.parse import urljoin
+
+from requests import Session
+from requests.adapters import HTTPAdapter, Retry
+
+
+class ServiceCRMException(Exception):
+    pass
+
+
+@dataclass
+class ServiceRequest:
+    """
+    Example:
+    {
+      "externalUserEmail": "marketplace@softwareone.com",
+      "externalUsername": "mpt@marketplace.com",
+      "requester": "Supplier.Portal",
+      "subService": "Service Activation",
+      "globalacademicExtUserId": "notapplicable",
+      "additionalInfo": "Service Activation",
+      "summary": "Request for Service Activation",
+      "title": "Request for Service Activation for SW-CCO-123-123",
+      "serviceType": "MarketPlaceServiceActivation"
+
+    }
+    """
+    externalUserEmail: str
```

Comment:
well, our guideline says that we use snake-case :-( Can we please try to follow it?